testing/src/network/sync.rs

```diff
@@ -105,7 +105,7 @@ async fn block_initiator_side() {
 
     // check the blocks have been added to the node's chain
     wait_until!(
-        5,
+        65,
```

Comment:
Might be good to comment on the test that it is expected to be long running because of the way sync currently works?